CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## UNRELEASED
+
+* Check for AWS_ROLE_ARN_ID in utils:connect_to_aws before attempting to
+assume a role using the STS service
```

Comment:
Could you rephrase this to say something like "Change AWS connection to use STS AssumeRole when AWS_ROLE_ARN_ID environment variable is specified in addition to just the profile being called "cross-account".